<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditInMainDialog</class>
 <widget class="QDialog" name="EditInMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <widget class="QPushButton" name="okButtonEdit">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButtonEdit">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="applyButtonEdit">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <item>
        <widget class="QLabel" name="RecipeNameLabelEdit">
         <property name="text">
          <string>Recipe name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="RecipeNameLineEdit">
         <property name="maxLength">
          <number>256</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="recipeLabelEdit">
         <property name="text">
          <string>Recipe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="RecipeTextEditEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="ingredientsLabelEdit">
         <property name="text">
          <string>Ingredients</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="IngredientsTableViewEdit"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <widget class="QLineEdit" name="ingredientNameLineEdit">
           <property name="maxLength">
            <number>256</number>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="nameLabelEdit">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_19">
         <item>
          <widget class="QLineEdit" name="QuantityLineEdit">
           <property name="maxLength">
            <number>256</number>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="quantityLabelEdit">
           <property name="text">
            <string>Quantity</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_20">
         <item>
          <widget class="QLineEdit" name="UnitLineEdit">
           <property name="maxLength">
            <number>256</number>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="unitLabelEdit">
           <property name="text">
            <string>Unit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="addButtonEdit">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modifyButtonEdit">
         <property name="text">
          <string>Modify</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButtonEdit">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
